scripts/apps/web-publisher/controllers/WebPublisherManagerController.js

```diff
@@ -0,0 +1,25 @@
+WebPublisherManagerController.$inject = ['$scope', 'api'];
+export function WebPublisherManagerController($scope, api) {
+	$scope.sites = ['content'];
+	$scope.manageSite = false;
+	
+    $scope.toogleCreateSite = function() {
+    	$scope.new = {};
+        $scope.manageSite = !$scope.manageSite;
+    };
+
+    $scope.save = function() {
+  //       content.createSite($scope.new)
```

Comment:
Let's not commit commented out code into the main branch please, it's a bad practice.